tests/Integration/Controller/ShowProductActionTest.php

```diff
@@ -187,19 +187,52 @@ public function itDisplaysAMappedProduct(): void
         $this->assertSelectorTextContains('h1.page-title', 'Kodak i2600 for Govt');
 
         $foundAttributes = $crawler->filter('.attribute');
-        $this->assertEquals(4, $foundAttributes->count());
+        $this->assertEquals(15, $foundAttributes->count());
 
-        $eanLabel = $foundAttributes->eq(0)->filter('.attribute__label')->text();
-        $this->assertEquals('Product UUID', $eanLabel);
+        $label0 = $foundAttributes->eq(0)->filter('.attribute__label')->text();
+        $this->assertEquals('Product UUID', $label0);
 
-        $eanLabel = $foundAttributes->eq(1)->filter('.attribute__label')->text();
-        $this->assertEquals('Title', $eanLabel);
+        $label1 = $foundAttributes->eq(1)->filter('.attribute__label')->text();
+        $this->assertEquals('SKU (Stock Keeping Unit)', $label1);
 
-        $eanLabel = $foundAttributes->eq(2)->filter('.attribute__label')->text();
-        $this->assertEquals('Description', $eanLabel);
+        $label2 = $foundAttributes->eq(2)->filter('.attribute__label')->text();
+        $this->assertEquals('Product name', $label2);
+
```

Comment:
IMO, we can reuse an unique variable `$label`, or even zero variable.

Reply:
I am not sure that zero variable is a good solution for readability.

But for the first proposition, IMO it's less source of error than your solution. For each test, you need to increment the "iterator" at three places.
But, it's a detail. We can debate later about this non primordial subject.

Reply:
Isn't a for-loop over an array of strings is the way ?

Reply:
done with a for-loop